asv_bench/benchmarks/groupby.py

```diff
@@ -591,12 +593,10 @@ class GroupByCythonAgg:
         [
             "sum",
             "prod",
-            # TODO: uncomment min/max
-            # Currently, min/max implemented very inefficiently
-            # because it re-uses the Window min/max kernel
-            # so it will time out ASVs
-            # "min",
-            # "max",
+            "min",
+            "max",
```

Comment:
min/max are already fixed?

Reply:
This was commented out by mistake. Ref: https://github.com/pandas-dev/pandas/commit/870a504af96a2ebcb89f2b09b110a9358f7771e7#r122434835